gammagl/layers/conv/mixhop_conv.py

```diff
@@ -0,0 +1,79 @@
+import tensorlayerx as tlx
+from gammagl.layers.conv import MessagePassing
+from gammagl import utils
```

Comment:
I think it's better to use relative paths, we can use "gammagl.layer.conv" because we already have the "gammagl" package installed and if we use relative paths we can run it even if we don't install it.

Reply:
RIgint.